docs/JSON-RPC-API/Using-JSON-RPC-API.md

```diff
@@ -25,14 +13,40 @@ Set the host to `0.0.0.0` to allow remote connections.
 
 !!! caution 
     Setting the host to 0.0.0.0 exposes the RPC connection on your node to any remote connection. In a 
-    production environment, ensure you are using a firewall to avoid exposing your node to the internet.  
+    production environment, ensure you use a firewall to avoid exposing your node to the internet.  
 
 Use the [--rpc-http-port](../Reference/Pantheon-CLI-Syntax.md#rpc-http-port) and [--rpc-ws-port](../Reference/Pantheon-CLI-Syntax.md#rpc-ws-port)
 options to specify the port on which the JSON-RPC listens. The default ports are: 
 
 * 8545 for HTTP
 * 8546 for WebSockets
 
+## Postman
+
+Use the button to import our collection of examples to [Postman](https://www.getpostman.com/). 
+
+[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/cffe1bc034b3ab139fa7)
+
+## Geth Console 
+
+To use the geth console with Pantheon: 
```

Comment:
May we explain why we want to use Geth console ?
Possible answers (on or more):
- we don't provide one in Pantheon as we think Geth one is working well
- it enables you to use JS directly in the console to create small script around calling the rpc method
- it makes Pantheon able to be managed with non interactive JS scripts.
- ...